src/pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -302,6 +302,7 @@ def __init__(self, extra_options):
                 "interstitial-diffusion limited",
                 "ec reaction limited",
                 "ec reaction limited (asymmetric)",
+                "Kolzenberg2020",
```

Comment:
```suggestion
                "VonKolzenberg2020",
```

I suggest using this naming, to match how we usually refer to parameters.

Reply:
Note that adding the options made some test fail (tests/unit/test_models/test_full_battery_models/test_base_battery_model.py). You just need to go into that test and add the new option in the long string that contains all model options.